CMakeLists.txt

```diff
@@ -317,6 +317,13 @@ set(LIBVNCCLIENT_SOURCES
     ${COMMON_DIR}/minilzo.c
 )
 
+if(JPEG_FOUND)
```

Comment:
As the encoder (and now the decoder as well) only works with libjpeg-turbo (am I right on this point?) the libjpeg differentiation in https://github.com/dcommander/libvncserver/blob/f9798b4309c31547e49cceaade43e46b6e71bd85/CMakeLists.txt#L59 can probably go away?

Reply:
No, they should both work with libjpeg as well.